apollo-compiler/src/main/kotlin/com/apollographql/apollo3/compiler/Options.kt

```diff
@@ -39,6 +39,43 @@ enum class TargetLanguage {
   KOTLIN_1_5,
 }
 
+enum class JavaNullableFieldStyle {
+  /**
+   * Fields will be generated with the same type whether they are nullable or not.
+   * This is the default value.
+   */
+  SIMPLE,
+
+  /**
+   * Fields will be generated as Apollo's `apollo.api.Optional<Type>` if nullable, or `Type` if not.
```

Comment:
It's a bit unfortunate but we have a super long package name...

```suggestion
   * Fields will be generated as Apollo's `com.apollographql.apollo3.api.Optional<Type>` if nullable, or `Type` if not.
```

Reply:
I think Copilot wrote this and I didn't notice it was wrong 🙈 😅